### PR TITLE
New package: TeamUTY.UndertaleYellow version 1.1.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132086)

### CODE REVIEW

File: manifests/t/TeamUTY/UndertaleYellow/1.1.0/TeamUTY.UndertaleYellow.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TeamUTY.UndertaleYellow
+PackageVersion: 1.1.0
+PackageLocale: en-US
+Publisher: Team Undertale Yellow
+PackageName: Undertale Yellow
```
Comment (stephengillie) [inline manifests/t/TeamUTY/UndertaleYellow/1.1.0/TeamUTY.UndertaleYellow.locale.en-US.yaml:8]:
```suggestion
PackageName: Undertale Yellow
PacakgeUrl: https://gamejolt.com/games/UndertaleYellow/136925
```

File: manifests/t/TeamUTY/UndertaleYellow/1.1.0/TeamUTY.UndertaleYellow.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TeamUTY.UndertaleYellow
+PackageVersion: 1.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Undertale Yellow v1_1\Undertale Yellow.exe
+Installers:
+- InstallerUrl: https://download.gamejolt.net/01564098c7597e3525a229ffe576b2ada2768ed91040c7489cf1fbf56568bb91,1703892925,7/data/games/2/175/136925/files/657b82d7d3859/undertale-yellow-v1.1.zip
```
Comment (stephengillie) [inline manifests/t/TeamUTY/UndertaleYellow/1.1.0/TeamUTY.UndertaleYellow.installer.yaml:11]: Hi @Exorcism0666,

I got a slightly different path: https://download.gamejolt.net/3defa062b0613b393767b440dcab79e0dbe7602701cad073a2538924628dd66d,1703955918,7/data/games/2/175/136925/files/657b82d7d3859/undertale-yellow-v1.1.zip

It's the same pattern, which implies these might be time-based URLs with the time component hashed. What do you think?

Comment (Exorcism0666) [reply manifests/t/TeamUTY/UndertaleYellow/1.1.0/TeamUTY.UndertaleYellow.installer.yaml:11]: I think so! but I don't know how to go about putting the link there :c

**Comment by Exorcism0666** [comment]
## For moderators

This account is bot account and belongs to CoolPlayLin. Any issue should call him up instead of assigning to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let CoolPlayLin know).

Have a good day!

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification. It has the write permission of this account's fork so that it can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Pull request closed without merging**